drivers/storage/ide/uniata/id_probe.cpp

```diff
@@ -1943,6 +1944,9 @@ UniataClaimLegacyPCIIDE(
     NTSTATUS status;
     PCM_RESOURCE_LIST resourceList = NULL;
     UNICODE_STRING devname;
+#ifdef __REACTOS__
```

Comment:
Move this block 1 line up?

Reply:
To group it together with the other resourceList?

Reply:
I wanted to have the `#ifdef` declaration block after the 'old' local variables, having it between the local variables seems ugly(er).
Changed to suit both our requirments

Reply:
No. Your new solution is worse to both: Your old solution and also worse than what Serge requested.
I do request that to be changed, and I think Serges proposal was ok.
I do reject your current solution as sicne you moved, now an ifdef guard is missing.

Reply:
> ugly(er)

Less "beautiful" as a ROS diff, but more "logical" as an upstream fix.

Reply:
> I do reject your current solution as since you moved, now an ifdef guard is missing.
Um I have moved one of the "old" declarations (resouceList) down so that it is bundled up with the other resourceList as was suggested.
The new declaration (oldResList) is still guarded by an `ifdef`. I'm not sure I understand the question here.

If variable ordering is a concern here (and I mean really?). If variable ordering is a concern here. Having a new variable between the old ones re-orders the variables anyways. I mean its variables